ui/components/institutional/confirm-remove-jwt-modal/__snapshots__/confirm-remove-jwt-modal.test.js.snap

```diff
@@ -1,155 +1,3 @@
-                        href="https://etherscan.io/address/0xaD6D458402F60fD3Bd25163575031ACDce07538D"
-                        rel="noopener noreferrer"
-                        target="_blank"
-                      >
-                        <span
-                          class="box mm-text mm-text--inherit box--flex-direction-row box--color-primary-default"
-                        >
-                          0xaD6...538D
-                          <span
-                            class="box mm-icon mm-icon--size-md box--margin-left-1 box--display-inline-block box--flex-direction-row box--color-primary-default"
-                            style="mask-image: url('./images/icons/undefined.svg');"
-                          />
-                        </span>
-                      </a>
-                      <div>
-                        <div
-                          aria-describedby="tippy-tooltip-1"
-                          class=""
-                          data-original-title="Copy to clipboard"
-                          data-tooltipped=""
-                          style="display: inline; background-color: transparent;"
-                          tabindex="0"
-                        >
-                          <button
-                            class="custody-account-list__item__clipboard"
-                          >
-                            <span
-                              class="box mm-icon mm-icon--size-md box--display-inline-block box--flex-direction-row box--color-icon-muted"
-                              style="mask-image: url('./images/icons/undefined.svg');"
-                            />
-                          </button>
-                        </div>
-                      </div>
-                    </span>
-                  </label>
-                  <div
-                    class="mm-box mm-box--display-flex mm-box--justify-content-space-between"
-                  >
-                    <label
-                      class="box mm-text mm-label mm-label--html-for mm-text--body-md mm-text--font-weight-medium box--display-flex box--flex-direction-row box--align-items-center box--color-text-default"
-                      for="address-0"
-                    />
-                  </div>
-                </div>
-              </div>
-            </div>
-          </div>
-        </div>
-      </div>
-    </div>
-    <div
-      class="modal-container__footer"
-    >
-      <button
-        class="button btn--rounded btn-secondary modal-container__footer-button"
-        role="button"
-        tabindex="0"
-      >
-        Nevermind
-      </button>
-      <button
-        class="button btn--rounded btn-primary modal-container__footer-button"
-        role="button"
-        tabindex="0"
-      >
-        Remove
-      </button>
-    </div>
-  </div>
-</div>
-`;
+exports[`Confirm Remove JWT should render correctly 1`] = `<div />`;
```

Comment:
Is this desired?  The whole of snapshot contents just got  removed.

Reply:
Yes, I removed the 'nevermind' button since the new modal now includes a cross icon at the top for closing it."Yes, I removed the nevermind button because the new modal has the cross icon on top of it